File: java-app/src/main/java/com/mercadolibre/java_app/congrats/CongratsContract.java
package com.mercadolibre.java_app.congrats;

import androidx.annotation.ColorRes;
import androidx.annotation.NonNull;

import com.mercadolibre.java_app.commons.ShippingOptionView;

public interface CongratsContract {
    public interface CongratsView extends ShippingOptionView {
        void showTitle(@NonNull String titleText, @ColorRes int color);
        void showFooter(@NonNull String footerText);
    }
}

File: kt-app/src/main/java/com/mercadolibre/melikotlinmeetup/congrats/CongratsContract.kt
package com.mercadolibre.melikotlinmeetup.congrats

import com.mercadolibre.melikotlinmeetup.commons.ShippingOptionView

interface CongratsView : ShippingOptionView {
    fun showTitle(titleText: String, color: Int)
    fun showFooter(footerText: String)
}